src/signature.rs

```diff
@@ -302,14 +302,15 @@ pub use ec::curve25519::ed25519::{
     ED25519,
 
     Ed25519KeyPair,
+    Ed25519KeyPairComponents,
```

Comment:
This should be exposed from `ring::signature::primitives` not from `ring::signature`.